packages/website/src/components/ListSidebar.tsx

```diff
@@ -0,0 +1,126 @@
+import { AnimatePresence, motion } from 'framer-motion';
+import Link from 'next/link';
+import { useState } from 'react';
+import { VscChevronDown, VscChevronRight } from 'react-icons/vsc';
+import type { ItemListProps } from './ItemSidebar';
+import type { getMembers } from '~/util/parse.server';
+
+export interface ListSidebarSectionProps {
+	members: ReturnType<typeof getMembers>;
+	selectedMember?: string | undefined;
+	title: string;
+}
+
+interface GroupedMembers {
+	Classes: ItemListProps['data']['members'];
+	Enums: ItemListProps['data']['members'];
+	Interfaces: ItemListProps['data']['members'];
+	Types: ItemListProps['data']['members'];
+	Variables: ItemListProps['data']['members'];
+}
+
+function groupMembers(members: ItemListProps['data']['members']): GroupedMembers {
+	const Classes: ItemListProps['data']['members'] = [];
+	const Enums: ItemListProps['data']['members'] = [];
+	const Interfaces: ItemListProps['data']['members'] = [];
+	const Types: ItemListProps['data']['members'] = [];
+	const Variables: ItemListProps['data']['members'] = [];
+
+	for (const member of members) {
+		switch (member.kind) {
+			case 'Class':
+				Classes.push(member);
+				break;
+			case 'Enum':
+				Enums.push(member);
+				break;
+			case 'Interface':
+				Interfaces.push(member);
+				break;
+			case 'TypeAlias':
+				Types.push(member);
+				break;
+			case 'Variable':
+				Variables.push(member);
+				break;
+			default:
+				break;
+		}
+	}
+
+	return { Classes, Enums, Interfaces, Types, Variables };
+}
+
+export function ListSidebarSection({ members, selectedMember, title }: ListSidebarSectionProps) {
+	const [showList, setShowList] = useState(true);
+
+	return (
+		<div>
+			<h3
+				className="flex items-center dark:text-white m-0 text-sm font-semibold gap-2"
+				onClick={() => setShowList(!showList)}
+			>
+				{showList ? <VscChevronDown size={20} /> : <VscChevronRight size={20} />}
+				{title}
+			</h3>
+			<AnimatePresence exitBeforeEnter initial={false}>
+				{showList ? (
+					<motion.div
+						className="ml-10 mt-2 space-y-3"
```

Comment:
```suggestion
						className="ml-7 mt-2 space-y-3"
```

To properly align with the section text.